vita3k/shader/src/usse_utilities.cpp

```diff
@@ -685,6 +685,9 @@ spv::Id shader::usse::utils::load(spv::Builder &b, const SpirvShaderParameters &
     const auto dest_comp_count = dest_mask_to_comp_count(dest_mask);
     const std::size_t size_comp = get_data_type_size(op.type);
 
+    const int num_comp_in_single_float = static_cast<int>(4 / size_comp);
+    const int base_add = shift_offset / num_comp_in_single_float;
```

Comment:
regarding these things, I checked the shader that render the girl in first level of Adventures of Mana. The disassembly definitely treats repeat offset in data type unit (pack instruction) .... Can ask @Zangetsu38  specifically for save